libraries/browser-tracker-core/src/tracker/index.ts

```diff
@@ -301,7 +301,8 @@ export function Tracker(
         trackerConfiguration.customHeaders ?? {},
         trackerConfiguration.withCredentials ?? true,
         trackerConfiguration.retryStatusCodes ?? [],
-        (trackerConfiguration.dontRetryStatusCodes ?? []).concat([400, 401, 403, 410, 422])
+        (trackerConfiguration.dontRetryStatusCodes ?? []).concat([400, 401, 403, 410, 422]),
+        trackerConfiguration.idService || ''
```

Comment:
Nit: the `|| ''` part is not necessary, right? It's an optional argument in the constructor.